juju/client/_client.py

```diff
@@ -3,20 +3,20 @@
 
 from juju.client._definitions import *
 
-from juju.client import _client1, _client2, _client3, _client4, _client5, _client6, _client7, _client8, _client9, _client10, _client12
+from juju.client import _client2, _client1, _client3, _client4, _client5, _client8, _client7, _client9, _client10, _client6, _client12
```

Comment:
Is the ordering intentional to mess with people that have OCD? ;-)

Reply:
I'm triggered EVERYTIME... I might work out why it does this.